COLLABORATOR_GUIDE.md

```diff
@@ -283,13 +282,15 @@ git reset --hard upstream/main
 ## video.js releases
 
 Releasing video.js is partially automated through various scripts.
-To do a release, you need a couple of things: npm access, GitHub personal access token.
```

Comment:
comment(non-blocking,no-change): technically, there isn't even a need for npm access to do a release anymore. Just write access to the repo!

Reply:
👍 updated line